benchmarks/benchmark_model.py

```diff
@@ -157,3 +157,10 @@ def to_dict(self):
     d["suite_name"] = self.suite_name
     d["model_name"] = self.model_name
     return d
+
+  @property
+  def default_precision_flag(self):
+    return None
+
+  def apply_env(self, process_env):
```

Comment:
Apply sounds to me like it changes the env vars. Maybe `extend_process_env`?

Reply:
Done.